src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sale = 0;
```

Comment:
I guess you mean "discount". Just create variables for big and small discounts, no need to reassign the variable later
![image](https://user-images.githubusercontent.com/96409563/234329325-a1c847b7-3e2c-4569-b314-23c283688f8c.png)